app/src/main/java/fr/free/nrw/commons/contributions/Contribution.kt

```diff
@@ -52,7 +53,8 @@ data class Contribution constructor(
         item: UploadItem,
         sessionManager: SessionManager,
         depictedItems: List<DepictedItem>,
-        categories: List<String>
+        categories: List<String>,
+        sha1: String
```

Comment:
Any reason to not use the same variable name `imageSHA1`?
In Java/Kotlin long variable names are not considered problematic :-)

Reply:
Thanks for reminding me! I have changed it to imageSHA1 and the variable names are consistent now.